sdk/keyvault/Azure.Security.KeyVault.Keys/CHANGELOG.md

```diff
@@ -4,12 +4,18 @@
 
 ### Features Added
 
+- Added `KeyProperties.HsmPlatform` to get the underlying HSM platform.
+
 ### Breaking Changes
 
 ### Bugs Fixed
 
+- `KeyRotationPolicyAction` performs case-insensitive comparisons since Key Vault and Managed HSM return different cases for "rotate".
```

Comment:
This was a server-side fix. No reason to mention it since it doesn't affect anyone's code nor did anyone (hopefully) notice.

Reply:
Removed 🙂